### PR TITLE
Swiftui readme update

### PR DESCRIPTION
**Pull request by ZvonimirMedak**
## Summary

This PR updates the Readme based on the recent changes we did for the 2.0.0 update

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [ ] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [X] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

### Description

Updated the Readme.md file to reflect the changes for 2.0.0 update

## Checklist

- [X] I have performed a self-review of my own code.
- [X] I have tested my changes, including edge cases.
- [X] I have added necessary tests for the changes introduced (if applicable).
- [X] I have updated the documentation to reflect my changes (if applicable).

## Additional notes

There could be some changes to the Readme if we go with the MacOS 13

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -3,9 +3,20 @@
 
 import PackageDescription
 
+// Excluding some Tools due to being tightly coupled to some unsupported frameworks on macOS
+#if os(macOS)
+let excludedSources: [String] = [
+    "Classes/CustomLocation",
+    "Classes/EmailSender"
+]
+#else
```
Comment (nikolamajcen) [inline Package.swift:12]: I would rather check if it is `iOS` due to potential changes needed e.g. if we introduce `tvOS` support.

File: README.md [hunk 2]
```diff
@@ -49,12 +52,12 @@ pod 'Sentinel'
 *Sentinel* is made of multiple subspecs:
 - `Core` which will install only the core features for the *Sentinel* to be usable
 - `UserDefaults` which will add the `UserDefaultsTool`
-- `EmailSendel` which will add the `EmailSenderTool`
+- `EmailSender` which will add the `EmailSenderTool`
 - `CustomLocation` which will add the `CustomLocationTool`
 - `TextEditing` which will add the `TextEditingTool`
-- `Default` - which will install `UserDefaults`, `Core`, `CustomLocation`, and `TextEditing`
+- `Default` - which will install `UserDefaults`, and `TextEditing`
```
Comment (nikolamajcen) [inline README.md:58]: Based on this list, I would recommend adding a section like; "Available tools" and list all of them with info about platform support and one sentence description.

And for CocoaPods and SPM, just add info if necessary - e.g., for CocoaPods it can stay as is, while notes about macOS support can be removed because that would be mentioned already.

File: README.md [hunk 3]
```diff
@@ -144,47 +199,95 @@ OptionSwitchItem(
...
 
+#### iOS only
+
+#### CustomLocationTool
+
+*CustomLocationTool* is used to change the current user's location. After changing the location, the application will have to be restarted.
+
+#### EmailSenderTool
+
 *EmailSenderTool* is used to let the user send emails with attachments from the app with ease.
```
Comment (nikolamajcen) [inline README.md:291]: As mentioned before, I would change a description of those and describe available tool for the developers (which are also available via Podspec).

**Review by nikolamajcen — CHANGES_REQUESTED** [review]
Nice work @ZvonimirMedak 👏  One step closer to the release of `v2.0.0` 🚀

I've added some comments, so please check them out (basically `README.md` changes). Everything else seems okay!

**Review by nikolamajcen — APPROVED** [review]
All good 👍
